datajunction-clients/python/datajunction/admin.py

```diff
@@ -0,0 +1,119 @@
+"""DataJunction main client module."""
+
+from typing import List, Optional
+
+from datajunction.builder import DJBuilder
+from datajunction.exceptions import DJClientException
+
+
+class DJAdmin(DJBuilder):  # pylint: disable=too-many-public-methods
+    """
+    Client class for DJ system administration.
+    """
+
+    #
+    # Data Catalogs
+    #
+    def list_catalogs(self) -> List[str]:
```

Comment:
I think users should be able to list catalogs as well. Although many organizations might only have a single catalog, it might still be helpful for first-timers